packages/html-webpack-plugin/README.md

```diff
@@ -0,0 +1,42 @@
+# @talend/html-webpack-plugin
+
+This plugin is to provide some default [hooks](https://github.com/jantimon/html-webpack-plugin#events) into `html-webpack-plugin`
+
+## options
+
+The plugin support the following options:
+
+| name | description |
```

Comment:
Don't forget to update this. 2 options missing `appLoaderIcon` and `versions`. And I think one was removed, `headerStyles`?